devicetypes/smartthings/zigbee-valve.src/zigbee-valve.groovy

```diff
@@ -160,19 +131,14 @@ def close() {
 
 def refresh() {
     log.debug "refresh called"
-    def cmds= []
+
+    def cmds = []
     cmds += zigbee.onOffRefresh()
-    cmds += zigbee.readAttribute(CLUSTER_BASIC, BASIC_ATTR_POWER_SOURCE)
     cmds += zigbee.onOffConfig()
+    cmds += zigbee.readAttribute(CLUSTER_BASIC, BASIC_ATTR_POWER_SOURCE)
     cmds += zigbee.configureReporting(CLUSTER_BASIC, BASIC_ATTR_POWER_SOURCE, DataType.ENUM8, 5, 21600, 1)
-    if (device.getDataValue("model") == "E253-KR0B0ZX-HA") {
-        cmds += zigbee.readAttribute(zigbee.IAS_ZONE_CLUSTER, zigbee.ATTRIBUTE_IAS_ZONE_STATUS)
-        cmds += zigbee.enrollResponse()
-        cmds += zigbee.configureReporting(zigbee.IAS_ZONE_CLUSTER, zigbee.ATTRIBUTE_IAS_ZONE_STATUS, DataType.BITMAP16, 0, 60 * 60, null)
-    } else {
-        cmds += zigbee.readAttribute(CLUSTER_POWER, POWER_ATTR_BATTERY_PERCENTAGE_REMAINING)
-        cmds += zigbee.configureReporting(CLUSTER_POWER, POWER_ATTR_BATTERY_PERCENTAGE_REMAINING, DataType.UINT8, 600, 21600, 1)
-    }
+    cmds += zigbee.readAttribute(CLUSTER_POWER, POWER_ATTR_BATTERY_PERCENTAGE_REMAINING)
```

Comment:
This should be higher up in the command list. We want to read all the attributes before configuring reporting so the current device status is reflected more quickly